lib/pdf/core/page.rb

```diff
@@ -140,6 +141,11 @@ def dimensions
         end
       end
 
+      def trim_dimensions
+        x1, y1, x2, y2 = dimensions
+        [x1 + bleed, y1 + bleed, x2 - bleed, y2 - bleed]
```

Comment:
It is not required to be the same on all sides but I wasn't completely sure of the dimensions in here without tests so I made the simplest possible thing.